app/views/tags/_grouped_admin_checkboxes.html.haml

```diff
@@ -0,0 +1,20 @@
+%div
+  - i=0
+  - alpha_tags.each_slice(6) do |gr|
```

Comment:
I've removed slice, no need to use column-count but I have been forced to change style as follows to avoid weird layout when having several lines for a given cell an next one only one, then instead of creating a new row it was just putting it below. 

Reviewing in Internet I found that you have to clear to force a new row and maintain the responsiveness

Try with this style, you'll notice 6 columns and reducing size of screen is respecting responsiveness for smaller size properly. Otherwise you get a weird layout.

under tags.css.scss:
.col-md-2:nth-child(7)  {
	clear: both;
}